buildkite/build_release.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -xueo pipefail
+trap 'echo build script failed at line $LINENO' ERR
+
+TODAY=$(date +'%Y-%m-%d')
+: ${BRANCH:=master}
+
+if [ ! -d installer ]; then
+    git clone https://github.com/dlang/installer.git installer
+fi
+pushd installer/create_dmd_release
+
+git reset --hard --quiet
+git clean -df --quiet
+if [ $BRANCH != master ] && [ $BRANCH != stable ] &&
+       ! git ls-remote --exit-code --heads origin $BRANCH > /dev/null; then
+    # use master as fallback base for feature branches
+    git fetch origin master --quiet
+else
+    git fetch origin $BRANCH --quiet
+fi
+git checkout FETCH_HEAD --quiet
+
+mkdir build
+LATEST=$(curl -fsSL http://downloads.dlang.org/releases/LATEST)
+rdmd build_all.d v$LATEST $BRANCH && success=yes || success=no
```

Comment:
So the idea is to upload everything that got build as success=no could mean that only one Box failed?
Maybe a good idea to add this as a comment.

Reply:
Good point, this was used in the old setup to uploads build logs.
So far we never uploaded partially succeeding builds.